.github/ISSUE_TEMPLATE/experimental_feature.md

```diff
@@ -1,31 +1,47 @@
 ---
 name: Experimental Feature Requirements
-about: Requirements for experimental features
+about: Requirements for an experimental feature
 ---
 
-# Experimental Feature Requirements
+# Experimental feature requirements
 
-This page lists the requirements for experimental features. Experimental features allow Istio work groups to collect more information
-on potential new functionality. This template should be completed before users try out any new experimental features. Please check off and 
-document the steps as they are completed.
+This page lists the requirements for an experimental feature. Completion of this template enables Istio work groups to collect
+information on potential new functionality. This template should be completed before users are exposed to any new
+experimental feature. Please complete this template during development.
 
-**Feature Name:** 
+This template shall be completed before users are exposed to any new experimetnal feature.
+
+# Mandatory fields for submission
+
+**Feature name:**
+
+**Primary lead:**
+
+**Short description:**
 
 --- 
 
-### Requirements: 
+# Requirements:
 
-- [ ] Feature is reviewed in a work group meeting
-- [ ] RFC has been written
+- [ ] Review of user stories in a work group meeting
+- [ ] RFC Authored
```

Comment:
Discussed this in Slack, but we have templates for design docs and RFCs in the Istio drive. We should link to them in the templates.

Reply:
agreed applied.